gm4_blast_furnaces/data/blast_furnaces/functions/handle_smelting.mcfunction

```diff
@@ -0,0 +1,9 @@
+# Called from check_structure.mcfunction
+
+# @s = @e[tag=gm4_bf_output, tag=gm4_bf_has_furnace, tag=gm4_bf_has_structure]
+
+# =========== MARK ADN DISPLAY ORE IFF IRON OR GOLD IS SMELTING ==========
+execute if block ^ ^ ^1 furnace{Items:[{Slot:0b,id:"minecraft:iron_ore"}]} run function blast_furnaces:ores/iron
+execute if block ^ ^ ^1 furnace{Items:[{Slot:0b,id:"minecraft:gold_ore"}]} run function blast_furnaces:ores/gold
```

Comment:
Blast furnaces should also double sand-glass and chorus-popped_chorus

Reply:
I did not know this, I'll add those.